rviz_default_plugins/src/rviz_default_plugins/tools/measure/measure_tool.cpp

```diff
@@ -58,6 +58,8 @@ namespace tools
 MeasureTool::MeasureTool()
 : is_line_started_(false), length_(-1)
 {
+  shortcut_key_ = 'n';
```

Comment:
Should all of these be part of the member initialization section? `: is_line_..., shurtcut_key_('n')`?

Reply:
The problem is that they are protected members of the `rviz_common::Tool` class, so I can't do that - I would have to provide another non-default constructor for Tool.
 
There is a comment for the Tool constructor saying that the `shortcut_key_` member should be initialized in the constructor.